README.md

```diff
@@ -17,7 +17,33 @@ If this proposal stands, we plan to extend the [Background Sync](https://wicg.gi
 specification to describe this ability as well, and move the document to the standardization track.
 
 ## Use cases
-  * todo
+  Many apps benefit from updating their state and content when the device has
```

Comment:
nit: the existing lines in this file have a 100 character line length limit, your additions follow 80 character limits. Please make that consistent.